Automattic/wp-calypso
# Domains: Remove site lists from upgrades actions

Part of #8726.

__This branch depends on #13094, which migrates `siteSelection` to use Redux state.__

### Testing
1. Go to http://calypso.localhost:3000/domains/manage.
2. Select your site which has a domain registration, a domain mapping and a site redirect (if you don't have a site redirect, you can purchase it from http://calypso.localhost:3000/domains/add/site-redirect).
3. Click `Change primary` button and pick your new primary domain.
4. Make sure it gets updated properly in the UI: url should change, site's card on the top left and primary domain on the list.
5. Repeat for all domain types.
6. Click on non-primary domain.
7. Click `Make Primary` button.
8. Click `Update Primary Domain`.
9. You should get redirect to the list of domains and the same set of changes as in (4) should happen.
10. Repeat for all domain types.

### Review

Note: CircleCI linting issues are inherited from #13094.
